pwnagotchi/plugins/default/wpa-sec.py

```diff
@@ -78,6 +91,12 @@ def on_loaded(self):
             logging.error("WPA_SEC: API-URL isn't set. Can't upload, no endpoint configured.")
             return
 
+        if 'whitelist' not in self.options:
+            self.options['whitelist'] = []
+
+        # remove special characters from whitelist APs to match on-disk format
+        self.options['whitelist'] = set(map(lambda x: re.sub(r'[^a-zA-Z0-9]', '', x), self.options['whitelist']))
```

Comment:
Add _ please

Reply:
@Evg33 Sure, will do. It also has to be fixed for OnlineHashCrack though. Should I open a new issue ?

Reply:
Also, allowing "_" would break the following:

`ssid, bssid = basename.split('_')`

Can you confirm that the format of your file is "My_Ap_aabbccddeeff" ?

Cheers,

ArnCo

Reply:
handshake files without '_'. It's lost.

Reply:
Alright, thanks dude ! Looks like we're good to go :smile: !